.github/workflows/ci.yml

```diff
@@ -103,6 +103,10 @@ jobs:
           if ((Get-FileHash 'sqliteodbc_w64.exe').Hash -ne "0df79be4a4412542839ebf405b20d95a7dfc803da0b0b6b0dc653d30dc82ee84") {Write "Hash mismatch"; Exit 1}
           ./sqliteodbc_w64.exe /S
         shell: powershell
+      - id: remove-strawberry-perl-tool-executables
+        run: |
+          Remove-Item -Path "C:\Strawberry\c\bin" -Recurse -Force
+        shell: pwsh
```

Comment:
Could you add a comment about why it's removed? We know now, but by the next time someone changes this piece of code, they may not remember all of the issues.

Reply:
Added comment in 99582f2591360093d29e94cd7202d28a2ed9f6b1